src/views/ModelPlan/CollaborationArea/Cards/ModelPlanCard/index.tsx

```diff
@@ -0,0 +1,251 @@
+import { TaskListStatusTag } from 'views/ModelPlan/TaskList/_components/TaskListItem';
+
+import './index.scss';
+
+type GetModelPlanTypes = GetModelPlanQuery['modelPlan'];
+type OperationalNeedsType = GetModelPlanQuery['modelPlan']['operationalNeeds'][0];
+
+type ModelPlanCardType = {
+  modelID: string;
+  setStatusMessage: (message: StatusMessageType) => void;
+};
+
+const ModelPlanCard = ({ modelID, setStatusMessage }: ModelPlanCardType) => {
+  const { t: collaborationAreaT } = useTranslation('collaborationArea');
+  const [isExportModalOpen, setIsExportModalOpen] = useState<boolean>(false);
+  const { data, loading } = useGetModelPlanQuery({
+    variables: {
+      id: modelID
+    }
+  });
+
+  const modelPlan =
+    data?.modelPlan ||
+    ({
+      __typename: 'ModelPlan',
+      id: modelID,
+      modelName: 'Test',
+      archived: false,
+      status: ModelStatus.PLAN_DRAFT,
+      modifiedDts: '',
+      modifiedByUserAccount: {
+        __typename: 'UserAccount',
+        commonName: ''
+      },
+      basics: {
+        __typename: 'PlanBasics',
+        id: '123',
+        status: TaskStatus.READY
+      },
+      generalCharacteristics: {
+        __typename: 'PlanGeneralCharacteristics',
+        id: '54234',
+        createdBy: 'John Doe',
+        createdDts: '',
+        modifiedBy: '',
+        modifiedDts: '',
+        readyForClearanceDts: '',
+        status: TaskStatus.IN_PROGRESS
+      },
+      participantsAndProviders: {
+        __typename: 'PlanParticipantsAndProviders',
+        id: '46246356',
+        createdBy: 'John Doe',
+        createdDts: '',
+        modifiedBy: '',
+        modifiedDts: '',
+        readyForClearanceDts: '',
+        status: TaskStatus.IN_PROGRESS
+      },
+      beneficiaries: {
+        __typename: 'PlanBeneficiaries',
+        id: '09865643',
+        createdBy: 'John Doe',
+        createdDts: '',
+        modifiedBy: '',
+        modifiedDts: '',
+        readyForClearanceDts: '',
+        status: TaskStatus.IN_PROGRESS
+      },
+      opsEvalAndLearning: {
+        __typename: 'PlanOpsEvalAndLearning',
+        id: '7865676',
+        createdBy: 'John Doe',
+        createdDts: '',
+        modifiedBy: '',
+        modifiedDts: '',
+        readyForClearanceDts: '',
+        status: TaskStatus.IN_PROGRESS
+      },
+      payments: {
+        __typename: 'PlanPayments',
+        id: '8756435235',
+        createdBy: 'John Doe',
+        createdDts: '',
+        modifiedBy: '',
+        modifiedDts: '',
+        readyForClearanceDts: '',
+        status: TaskStatus.IN_PROGRESS
+      },
+      operationalNeeds: [],
+      taskListStatus: TaskStatus.READY,
+      isFavorite: true,
+      collaborators: [],
+      documents: [],
+      crs: [],
+      tdls: [],
+      discussions: [],
+      prepareForClearance: {
+        __typename: 'PrepareForClearance',
+        status: PrepareForClearanceStatus.IN_PROGRESS,
+        modifiedDts: ''
+      }
+    } as GetModelPlanTypes);
```

Comment:
I think a better way of ensuring we handle undefined data correctly will be to conditionally render the component if data doesn't exist.  This ^ way is better suited if we still need to render the component, but have conditonal parts of it that we don't want to render if they dont exist.  Lets pair on what this looks like.  He is an example of how it might look:

```
const modelPlan = data?.modelPlan;

  // Returns the number of sections that have been started (i.e. not in 'READY' status)
  const sectionStartedCounter = useMemo(() => {
    if (loading) return 0;

    const sections = [
      modelPlan?.basics.status,
      modelPlan?.generalCharacteristics.status,
      modelPlan?.participantsAndProviders.status,
      modelPlan?.beneficiaries.status,
      modelPlan?.opsEvalAndLearning.status,
      modelPlan?.payments.status,
      getITSolutionsStatus(modelPlan?.operationalNeeds ?? [])
    ];

    return sections.filter(status => status !== TaskStatus.READY).length;
  }, [
    loading,
    modelPlan?.basics?.status,
    modelPlan?.generalCharacteristics?.status,
    modelPlan?.participantsAndProviders?.status,
    modelPlan?.beneficiaries?.status,
    modelPlan?.opsEvalAndLearning?.status,
    modelPlan?.payments?.status,
    modelPlan?.operationalNeeds
  ]);

  if (loading) return <Spinner />;

  if (!modelPlan) return null;

  const { modifiedDts, modifiedByUserAccount, taskListStatus } =
    modelPlan || {};
```